lmdeploy/turbomind/deploy/target_model/base.py

```diff
@@ -195,9 +196,15 @@ def get_config(self, cfg: TurbomindModelConfig) -> TurbomindModelConfig:
             emb = bin.tok_embeddings()
             if emb is not None:
                 _vocab_size, dim = emb.shape
-                head_num = dim // cfg.size_per_head
+                # legacy, may get removed
+                head_num = final_cfg['head_num'] or dim // cfg.size_per_head
```

Comment:
I think we can remove "or dim // cfg.size_per_head"

Reply:
I don't know if it is ok to remove `dim // cfg.size_per_head`.

Reply:
It's fine, although I don't remember why I did that 😂